io.openems.backend.common/src/io/openems/backend/common/debugcycle/DebugLoggable.java

```diff
@@ -0,0 +1,26 @@
+package io.openems.backend.common.debugcycle;
+
+import java.util.Map;
+
+import com.google.gson.JsonElement;
+
+import io.openems.backend.common.timedata.Timedata;
+
+public interface DebugLoggable {
```

Comment:
Should be merged with AbstractOpenemsBackendComponent eventually, to have the same concept as on Edge. (https://github.com/OpenEMS/openems/blob/develop/io.openems.backend.common/src/io/openems/backend/common/component/AbstractOpenemsBackendComponent.java)